pre-docs/bolt_installing.md

```diff
@@ -10,6 +10,10 @@ Download from https://downloads.puppet.com/windows/puppet5/puppet-bolt-x64-lates
 
 ## Install Bolt on Mac OS X
 
+### Install with Brew
+`brew cask install puppetlabs/puppet/puppet-bolt`
+
+### Install from Source
```

Comment:
I'm not sure if "Install from Source" is the right heading here?

Reply:
Install packages manually?